A3A/addons/maps/Stringtable.xml

```diff
@@ -444,101 +498,122 @@
                 <Russian>2 мин</Russian>
                 <Korean>2분</Korean>
+                <French>2 min</French>
             </Key>
             <Key ID="STR_antistasi_mission_info_journal_autoLoadLastGame_3min">
                 <Original>3 min</Original>
                 <Russian>3 мин</Russian>
                 <Korean>3분</Korean>
+                <French>3 min</French>
             </Key>
             <Key ID="STR_antistasi_mission_info_journal_autoLoadLastGame_5min">
                 <Original>5 min</Original>
                 <Russian>5 мин</Russian>
                 <Korean>5분</Korean>
+                <French>5 min</French>
             </Key>
             <Key ID="STR_antistasi_mission_info_journal_autoLoadLastGame_10min">
                 <Original>10 min</Original>
                 <Russian>10 мин</Russian>
                 <Korean>10분</Korean>
+                <French>10 min</French>
             </Key>
             <Key ID="STR_antistasi_mission_info_journal_LogLevel_name">
                 <Original>Logging Level during setup</Original>
                 <Russian>Уровень логгирования</Russian>
                 <Korean>설정 중 로그 작성 수준</Korean>
+                <French>Niveau d'enregistrement</French>
             </Key>
             <Key ID="STR_antistasi_mission_info_journal_LogLevel_Error">
                 <Original>Error</Original>
                 <Russian>Ошибки</Russian>
                 <Korean>오류</Korean>
+                <French>Erreurs</French>
             </Key>
             <Key ID="STR_antistasi_mission_info_journal_LogLevel_Info">
                 <Original>Info</Original>
                 <Russian>Информация</Russian>
                 <Korean>정보</Korean>
+                <French>Informations</French>
             </Key>
             <Key ID="STR_antistasi_mission_info_journal_LogLevel_Debug">
                 <Original>Debug</Original>
                 <Russian>Дебаг</Russian>
                 <Korean>디버그</Korean>
+                <French>Débogage</French>
             </Key>
             <Key ID="STR_antistasi_mission_info_journal_LogLevel_Verbose">
                 <Original>Verbose</Original>
                 <Russian>Подробный</Russian>
                 <Korean>세부 정보</Korean>
+                <French>Détaillé</French>
             </Key>
             <Key ID="STR_antistasi_mission_info_journal_A3A_logDebugConsole_name">
                 <Original>Log debug console use during setup</Original>
                 <Russian>Логгировать использование дебаг консоли</Russian>
                 <Korean>설정 중 디버그 콘솔 사용 기록하기</Korean>
+                <French>Utilisation de la console de débogage pendant l'installation</French>
             </Key>
             <Key ID="STR_antistasi_mission_info_journal_A3A_logDebugConsole_None">
                 <Original>None</Original>
                 <Russian>Не логировать никого</Russian>
                 <Korean>없음</Korean>
+                <French>N'enregistrez personne
+</French>
```

Comment:
```suggestion
                <French>N'enregistrez personne</French>
```

Reply:
thank you, it's not good to work at night